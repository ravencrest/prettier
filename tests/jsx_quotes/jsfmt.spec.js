run_spec(__dirname, ["flow", "typescript"], { jsxSingleQuote: true });
run_spec(__dirname, ["flow", "typescript"], { jsxSingleQuote: false });
