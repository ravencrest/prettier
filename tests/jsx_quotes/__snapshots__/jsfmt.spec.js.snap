// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test.js 1`] = `
<SimpleComponent
  str0="abc"
  str1='abc'
  str2="a'b'c"
  str3='a"b"c'
/>
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
<SimpleComponent str0="abc" str1="abc" str2="a&apos;b&apos;c" str3="a"b"c" />;

`;

exports[`test.js 2`] = `
<SimpleComponent
  str0="abc"
  str1='abc'
  str2="a'b'c"
  str3='a"b"c'
/>
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
<SimpleComponent str0="abc" str1="abc" str2="a'b'c" str3="a&quot;b&quot;c" />;

`;
